translations/story/50/1003/515 (カイチョーみたいな勝負服).json

```
@@ -1393,7 +1393,7 @@
             "jpName": "エアグルーヴ",
             "enName": "Air Groove",
             "jpText": "よかったな。掃除で好きなだけ駆け回れるぞ。\r\nその上、会長も見守ってくださるそうだ。",
-            "enText": "Good for you. You can run around as much as \nyou want cleaning up. Besides, the chairman \nof the board will be watching over us.",
+            "enText": "Good for you. You can run around as much as \nyou want cleaning up. Besides, the chairman \nwill be watching over us.",
```

```suggestion
            "enText": "Good for you. You can run around as much as \nyou want cleaning up. Besides, the president\nwill be watching over us.",
```
Also Akikawa is generally referred to as chair(wo)man so it can get confusing.